testgres/node.py

```diff
@@ -667,13 +667,15 @@ def dump(self, dbname, username=None, filename=None):
             dbname: database name to connect to.
             username: database user name.
             filename: output file.
+            formate: format argument p/c/d/t
 
         Returns:
             Path to a file containing dump.
         """
 
         # Set default arguments
         username = username or _default_username()
+        formate = formate or "p"
```

Comment:
"p" should be replaced with a const defined in consts.py

Reply:
done